src/Graph.ts

```diff
@@ -73,3 +73,17 @@ function isAnExit(board: Board, x: number, y: number) {
     y === lastRowIndex
   );
 }
+
+export const graphToBoard = (graph: Graph): Board => {
+  let board: Board = [[]];
```

Comment:
In this case, I suppose you can declare board as `const` because pushing into an array doesn't reassign this variable.

Reply:
Of course, it's done